content/_snippets/rippled-api-links.md

```diff
@@ -7,6 +7,7 @@
 [Currency Code]: currency-formats.html#currency-codes
```

Comment:
Comments on this file should go to #553, not here. (This file will leave the diff after I merge that PR and rebase.)